vulnerabilities/package_managers.py

```diff
@@ -217,12 +265,15 @@ class MavenVersionAPI(VersionAPI):
     async def load_api(self, pkg_set):
         async with client_session() as session:
             await asyncio.gather(
-                *[self.fetch(pkg, session) for pkg in pkg_set if pkg not in self.cache]
+                *[
+                    self.fetch(pkg, session)
+                    for pkg in pkg_set
+                    if pkg not in self.cache and "camel" not in pkg
+                ]
             )
 
     async def fetch(self, pkg, session) -> None:
-        artifact_comps = pkg.split(":")
-        endpoint = self.artifact_url(artifact_comps)
+        endpoint = self.artifact_url(pkg)
```

Comment:
Do not stop using the `maven-metadata.xml` file to get versions. Instead add a new function to parser the HTTP directory listing HTML page if available. (It could very well be absent). Do not try to fetch metadata at the groupid level